static/app/gettingStartedDocs/javascript/svelte.tsx

```diff
@@ -208,10 +215,16 @@ const onboarding: OnboardingConfig = {
         {
           code: [
             {
-              label: 'JavaScript',
-              value: 'javascript',
-              language: 'javascript',
-              code: getVerifySnippet(),
+              label: 'Svelte v5',
+              value: 'svelte v5',
+              language: 'html',
+              code: getVerifySnippet(true),
+            },
+            {
+              label: 'Svelte v3/v4',
+              value: 'svelte v3/v4',
+              language: 'html',
+              code: getVerifySnippet(false),
```

Comment:
Using `language: html` here yields nicer but still somewhat broken results for the .svelte snippet.
![Bildschirmfoto 2025-03-31 um 14 36 29](https://github.com/user-attachments/assets/37a944a4-796a-4bd4-beb3-29e5c089a6ca)